table.go

```diff
@@ -28,9 +28,13 @@ type TableFamily byte
 
 // Possible TableFamily values.
 const (
-	TableFamilyIPv4   TableFamily = unix.AF_INET
-	TableFamilyIPv6   TableFamily = unix.AF_INET6
-	TableFamilyBridge TableFamily = unix.AF_BRIDGE
+	TableFamilyINet   TableFamily = unix.NFPROTO_INET
+	TableFamilyIPv4   TableFamily = unix.NFPROTO_IPV4
+	TableFamilyIPv6   TableFamily = unix.NFPROTO_IPV6
+	TableFamilyARP    TableFamily = unix.NFPROTO_ARP
+	TableFamilyNetdev TableFamily = unix.NFPROTO_NETDEV
+	TableFamilyBridge TableFamily = unix.NFPROTO_BRIDGE
+	TableFamilyDECNet TableFamily = unix.NFPROTO_DECNET
```

Comment:
Do we really need Decnet? I think we should not bring everything which Linux has accumulated for past almost 30 years, especially since lots of people reported not being able to run it with 4.X kernel.

Reply:
Sure, fine by me.